bentoml/_internal/server/grpc_app.py

```diff
@@ -110,10 +110,24 @@ def options(
 
     @property
     def interceptors(self) -> list[aio.ServerInterceptor]:
+        from opentelemetry import trace
+        from opentelemetry.sdk.trace.export import ConsoleSpanExporter
+        from opentelemetry.sdk.trace.export import SimpleSpanProcessor
+
         from .grpc.interceptors import ExceptionHandlerInterceptor
+        from .grpc.interceptors.access_log import AccessLogInterceptor
+
+        tracer_provider = BentoMLContainer.tracer_provider.get()
+        trace.set_tracer_provider(tracer_provider)
+        trace.get_tracer_provider().add_span_processor(
+            SimpleSpanProcessor(ConsoleSpanExporter())
+        )
 
-        # # TODO: add access log, tracing, prometheus interceptors.
-        interceptors: list[aio.ServerInterceptor] = [ExceptionHandlerInterceptor()]
+        # TODO: prometheus interceptors.
+        interceptors: list[aio.ServerInterceptor] = [
+            AccessLogInterceptor(),
+            ExceptionHandlerInterceptor(),
```

Comment:
```suggestion
            ExceptionHandlerInterceptor(),
            AccessLogInterceptor(),
```
swap this two around, order matters.